### PR TITLE
Documentation for ChannelNavigation & EPG

### PR DESCRIPTION
**Pull request by altran-jonathan-bijos**
Please check if the documentation is clear and/or if it needs more information

### CODE REVIEW

File: Marketplace/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+
+## Channel Navigation
+
+First, set the product configuration on the viewcontroller with it's **name** and **color**
+```swift
+private var config = ChannelNavigationProductConfig(name: "project-name", color: UIColor.projectColor)
+```
+
+Then, instantiate `channelNavigation` on the viewcontroller you wish to present this feature and set the configuration just created and set the **output delegate** to `self`
```
Comment (barbaracsilveira) [inline Marketplace/README.md]: Could you reduce the use of 'and' in the sentence?

File: Marketplace/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```swift
+private lazy var channelNavigation: ChannelNavigation = ChannelNavigation(config: config, output: self)
+```
+
+This feature has a protocol implementation that needs to be set on the viewcontroller
```
Comment (barbaracsilveira) [inline Marketplace/README.md]: What do you think of:
"This feature has a protocol implementation that needs to be set in viewcontroller"

File: Marketplace/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    super.viewDidLoad()
+    channelNavigation.present(in: self)
+}
+```
+
+And that's it. The rest of the configuration can be set where needed and the configuration can also be changed if the permissions change.
+
+## EPG
+
+First, set the product configuration on the viewcontroller with the **project name**,  **project color** and **user state**
```
Comment (barbaracsilveira) [inline Marketplace/README.md]: Could you change "project-name" to "tenant-name" and UIColor.projectColor to UIColor.productColor, please?

File: Marketplace/README.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
+# Marketplace
+
+
+## Channel Navigation
+
+First, set the product configuration on the viewcontroller with it's **name** and **color**
+```swift
+private var config = ChannelNavigationProductConfig(name: "project-name", color: UIColor.projectColor)
```
Comment (barbaracsilveira) [inline Marketplace/README.md]: Could you change "project-name" to "tenant-name" and UIColor.projectColor to UIColor.productColor, please?

File: Marketplace/README.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+```swift
+private var config = EPGProductConfig(name: "project-name", userState: .loggedOut)
+```
+
+Then, instantiate `epg` on the viewcontroller you wish to present this feature and set the configuration just created and set the **output delegate** to `self`
+```swift
+private lazy var epg: EPG = EPG(config: config, output: self)
+```
+
+The viewcontroller has to implement `EPGDelegate`
```
Comment (barbaracsilveira) [inline Marketplace/README.md]: Could you mention the ChannelNavigationDelegate like this too, please?

File: Marketplace/README.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+Then, instantiate `channelNavigation` on the viewcontroller you wish to present this feature and set the configuration just created and set the **output delegate** to `self`
+```swift
+private lazy var channelNavigation: ChannelNavigation = ChannelNavigation(config: config, output: self)
+```
+
+This feature has a protocol implementation that needs to be set on the viewcontroller
+```swift
+public protocol ChannelNavigationProtocol {
```
Comment (barbaracsilveira) [inline Marketplace/README.md]: Could you mention the EPGProtocol like this too, please?

File: Marketplace/README.md (hunk 2 above)
Comment (altran-jonathan-bijos) [reply Marketplace/README.md]: I'll change it to UIViewController to avoid confusion with with variable names and types

**Pull request merged**